euphonic/plot.py

```diff
@@ -78,11 +78,10 @@ def plot_1d_to_axis(spectra: Union[Spectrum1D, Spectrum1DCollection],
             label = None
             if x0 == 0:
                 if labels is None:
-                    if spectrum.metadata is not None:
-                        try:
-                            label = spectrum.metadata['label']
-                        except KeyError:
-                            pass
+                    try:
+                        label = spectrum.metadata['label']
+                    except KeyError:
+                        pass
```

Comment:
Can drop L78 and replace this try/except block with `label = spectrum.metadata.get('label')`.